lib/routes/account.js

```diff
@@ -484,13 +501,17 @@ module.exports = function (
                   uid: emailRecord.uid,
                   kA: emailRecord.kA,
                   wrapKb: wrapKb,
-                  emailVerified: emailRecord.emailVerified
+                  emailVerified: emailRecord.emailVerified,
+                  tokenVerificationId: tokenVerificationId,
+                  tokenVerified: false
                 })
               }
             )
             .then(
               function (keyFetchToken) {
                 response.keyFetchToken = keyFetchToken.data.toString('hex')
+                response.challengeMethod = 'email';
+                response.challengeReason = 'signin';
```

Comment:
The `challengeReason` type depends on the status of the account. If the email has been previously verified, `challengeReason` should be `signin`, if the email has yet to be verified and the user is signing in with an account that was never verified, the `challengeReason` should be `signup`.